assets/javascripts/ws_console.js

```diff
@@ -83,6 +83,21 @@ function establishWebSocketConnection() {
   };
 }
 
+function submitWebSocketCommand(event) {
+  if (event) {
+    event.preventDefault();
+  }
+  var msg = document.getElementById('msg');
+  var command = msg.value;
+  if (!window.ws || (window.wsUsingProxy && !window.wsProxyConnectionConcluded)) {
+    logStatus("Can't send command, no ws connection opened! Will try to send when connection has been restored.");
```

Comment:
Might be good to have the msg in there in case this goes wrong and one wonders which one it was?

```suggestion
    logStatus("Can't send ' + command + ', no ws connection opened! Will try to send when connection has been restored.");
```

Reply:
I only moved this code. Besides, the concrete message doesn't matter at all for this error condition (only the overall connection status).

Reply:
Fair enough 👍🏾